release.sh

```diff
@@ -126,6 +131,25 @@ for check in "${CHECKS[@]}"; do
         "cmd/$check" --push
 
     log_msg "Builded image $check:[${IMAGE_TAGS[*]}]"
+
+    if [ -f "cmd/$check/vulcan.yaml" ]; then
+        log_msg "Testing image $check"
+        vulcan-local -i "$check" -c "cmd/$check/vulcan.yaml" -s CRITICAL -checktypes checktypes.json -l DEBUG -r - 2> error.log 1> r.json || true
+        STATUS=$(jq < r.json -r '.[].status' | tr '\n' ',')
+        if [[ ! $STATUS =~ ^(FINISHED|INCONCLUSIVE|,)+$ ]]; then
```

Comment:
Probably INCONCLUSIVE should't be a success state, but hard to generate a working test for some checks (i.e. prowler)